### PR TITLE
Add libbluetooth dev deps to add python support to bluetooth sockets

### PR DESCRIPTION
**Pull request by sergioisidoro**
### ⚠️ Disclaimer: I'm new around here ⚠️

I was playing around with python and BLE, and stumbled upon this old issue. 
I decided to give it a try

https://github.com/ContinuumIO/anaconda-issues/issues/1035

Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by sergioisidoro** [comment]
Ok, this is clearly a bit out of my reach as a beginner right now :) 

So far, as I could gather the deps are`libbluetooth-dev` in debian, `bluez-dev` in alpine, and I have very little knowledge about windowns platform.

If anyone would point me in the right direction I might still try, but I'm a bit lost right now.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -92,6 +92,7 @@ requirements:
     - {{ compiler('cxx') }}
     - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
     - {{ cdt('libx11-devel') }}          # [linux]
+    - {{ cdt('libbluetooth-dev') }}      # [not win]
```
Comment (hmaarrfk) [inline recipe/meta.yaml]:
```suggestion
    - {{ cdt('libbluetooth-dev') }}      # [linux]
```

**Comment by hmaarrfk** [comment]
ohhh, this one is hard. one issue I see happening is that you will have to  think about what works for `cos6` and that might not be what works for `cos7`, which means that you might not be able to install your `cos6` package on ubuntu.

You might need to add things like cdts for the things you need. Check staged recipes for how to do that.

Do you need to change anything for OSX?

**Comment by scopatz** [comment]
Yes, this need cos7 to work, which we don't have yet

File: recipe/meta.yaml (hunk 1 above)
Comment (sergioisidoro) [reply recipe/meta.yaml]: This is something I found out later. it's `libbluetooth-dev` for debian / ubuntu, but `bluez-dev` for things like alpine distributions. So I have no idea on how to specify things on a per distro.

**Comment by sergioisidoro** [comment]
> Do you need to change anything for OSX?

I'm not really sure... As far as I could gather, python needs to be built with `bluetooth.h` so that `AF_BLUETOOTH` doesn't at least throw an undefined error. Apple seems to have some remainders of documentation for that header, and things seemed to work on my machine, if I recall correctly 

https://developer.apple.com/documentation/kernel/bluetooth_h?language=objc

**Comment by hmaarrfk** [comment]
This is close to being possible. I think there is going to be a cos6->cos7 migration soon, do you want to open an issue about this?

**Pull request closed without merging**